keyboards/keebio/bamfk1/keymaps/via/rules.mk

```diff
@@ -1,2 +1,5 @@
+TRI_LAYER_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
+MOUSEKEY_ENABLE = no
```

Comment:
Its generally wise to keep this enabled where possible, and trim firmware size elsewhere
```suggestion
```